docs/examples/nats.md

```diff
@@ -0,0 +1,69 @@
+# Nats
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for nats.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the nats module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/examples/nats
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a nats container](../../examples/nats/nats.go)
+<!--/codeinclude-->
+
+<!--codeinclude-->
+[Test for a nats container](../../examples/nats/nats_test.go)
+<!--/codeinclude-->
+
+## Module reference
```

Comment:
One last thing that I did not realise until now: this part of the docs, although created by the code generation tool, are for modules, not for example modules, so I think you can remove them. From this line to EOF. The same applies to https://github.com/testcontainers/testcontainers-go/pull/1190/files#diff-80fa2472c79436cc57568b167327c6e4bbdff148f17809d7da720a81c3b8d708R3 (the line including the release version)

Once we convert the example into a module, we will need them back